lms/static/scripts/frontend_apps/components/OAuth2RedirectErrorApp.js

```diff
@@ -51,77 +51,62 @@ export default function OAuth2RedirectErrorApp({ location = window.location }) {
   const retry = () => {
     location.href = /** @type {string} */ (authUrl);
   };
-
-  const buttons = [];
-
-  if (authUrl) {
-    buttons.push(
-      <LabeledButton
-        key="try-again"
-        onClick={retry}
-        data-testid="try-again"
-        variant="primary"
-      >
-        Try again
-      </LabeledButton>
-    );
```

Comment:
`ErrorModal`'s support for `onRetry` and `retryLabel` obviates the need to do this here.